assets/scss/base/form/_hint.scss

```diff
@@ -73,11 +73,15 @@ List
 
 A Form hint list. The form hint list can also be used in conjunction with `formHintHelper.js` for instance if you
 had a password text input and wanted to check the hints have been correctly matched.
+Optionally wrap the `.form-hint-list-item__indicator` element in a `.form-hint-list-item__primary` element to give a
```

Comment:
Sorry what I mean is **"If you wish to indent multiple lines of text with the same indent as used with an indicator then"** > wrap the `.form-hint-list-item__indicator` element in...